### PR TITLE
FEAT: add edit profile feature

### PR DESCRIPTION
**Pull request by agrawalshanu708**
Added a new feature -> Edit profile
User can Edit their profile details
image

### CODE REVIEW

File: src/comps/Profile/UserDetails/EditProfileModal.jsx [hunk 1]
```diff
@@ -36,31 +40,68 @@ function EditProfileModal() {
...
+                bio: e.target.value,
+              }))
+            }
           />
           <Button
             sx={{ display: "block", marginTop: "1rem" }}
             open={open}
-            onClick={() => setOpen(false)}
+            onClick={() => {
+              console.log(userProfileDetails);
```
Comment (sashaboi) [inline src/comps/Profile/UserDetails/EditProfileModal.jsx:102]: you could remove logs later

File: src/comps/Profile/UserDetails/UserDetails.jsx [hunk 2]
```diff
@@ -9,13 +9,20 @@ import PhoneAndroidOutlinedIcon from "@mui/icons-material/PhoneAndroidOutlined";
...
 
 const UserDetails = () => {
-  const[personalPost,setPersonalPost] = useState([])
-  const { allPost } = useSelector((store) => store.posts);
+  const [personalPost, setPersonalPost] = useState([]);
+  const [userProfileDetails, setUserProfileDetails] = useState({
+    profession: "software developer",
+    contactNumber: "9424431504",
+    website: "shanuagrawal.com",
+    bio: "I am a software developer who is well versed with JavaScript/ES6+ , ReactJS & Redux and an intermediate in UI/UX concepts. I can build web apps and can write modular code for better results. I am always eager to learn new technologies and methodologies",
```
Comment (sashaboi) [inline src/comps/Profile/UserDetails/UserDetails.jsx:20]: try to keep data in separate files and import later

**Review by sashaboi — APPROVED** [review]
looks good to merge !

**Review by MonikaJhingan — APPROVED** [review]
Looks goof to be merged.

**Pull request merged**